src/components/InfiniteLoading.vue

```diff
@@ -43,6 +43,7 @@ const stateHandler: StateHandler = {
     state.value = "loading";
   },
   async loaded() {
+    params.firstload = false;
```

Comment:
regarding the documentation, firstload is used for a different scope.
```
This property is used to specify weither you want the component to handle first load or not
```

Create an internal variable to store it.